Install.md

```diff
@@ -121,19 +108,28 @@ This guide helps you build and install Snort.
     sudo cp -r snort-2.9.8.3/simple-etc /etc/snort
     sudo mkdir /usr/local/lib/snort_dynamicrules
     ```
+2. Add snort to path (change /opt/snort if the install path is different)
+    ```
+    export PATH=$PATH:/opt/snort/bin"
+    ```
     
-2. Run openNetVM manager. To install openNetVM, refer to this [guide][onvm-install].
+3. Run openNetVM manager. To install openNetVM, refer to this [guide][onvm-install].
     ```sh
-    cd openNetVM-dev/onvm
-    ./go.sh 0,1,2,3,4 3 -v 0x7f000000000
+    cd openNetVM/onvm
+    ./go.sh 0,1,2,3 3 0xF0 -v 0x7f000000000 -s stdout
     ```
-3. Run Snort.
+4. Run Snort.
     ```sh
     sudo snort -A console -Q -c /etc/snort/snort.conf -i dpdk0:dpdk1 -N --alert-before-pass --daq-var netvm_args="-l 5 -n 3 --proc-type=secondary -- -r 1 -- -d 4"
     ```
+    If the above does not work then try:
+    ```
+    which snort
+    sudo `which snort` -A console -Q -c /etc/snort/snort.conf -i dpdk0:dpdk1 -N --alert-before-pass --daq-var netvm_args="-l 5 -n 3 --proc-type=secondary -- -r 1 -- -d 4"
```

Comment:
Running into an issue here:
<img width="625" alt="Screen Shot 2019-10-31 at 2 42 10 PM" src="https://user-images.githubusercontent.com/32916582/67976591-c6f04700-fbec-11e9-8a3a-ad5a6cc65d6e.png">

Did you run into something similar?

Reply:
Did not come accross this issue.. I'll look into what might be causing this

Reply:
Looks like when using --disable-shared, snort does not build the dynamic_preprocessors. I've edited the snort.conf to disable these dynamic_preprocessors.